FusionMCPBridge/tests/conftest.py

```
@@ -0,0 +1,463 @@
+        
+        data = response.json()
+        if data.get("error") and "MANUFACTURE" in data.get("message", "").upper():
+            pytest.skip("MANUFACTURE workspace not active")
+    except requests.exceptions.RequestException:
+        pytest.skip("Could not verify MANUFACTURE workspace availability")
+    
+    return True
+
+
+@pytest.fixture
+def design_document_required(bridge_available):
```

Ruff (0.14.10)

Unused function argument: `bridge_available`

(ARG001)